pyramid_oereb/contrib/scripts/__init__.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import optparse
```

Comment:
optparse is deprecated, you should use argparse...
see: https://docs.python.org/3/library/optparse.html